sql/moz-fx-data-shared-prod/jira_service_desk/dataset_metadata.yaml

```
@@ -4,7 +4,13 @@ description: |-
 dataset_base_acl: view
```

By default DE has write access to datasets (which means read access too). To remove that as well you need to set these to the restricted dataset variants
```suggestion
dataset_base_acl: view_restricted
```
Note that this means you as a DE will also not have direct write access. You can add it back by granting write access to `workgroup:service-desk/data-developers` in `workgroup_access`